lib/cib/cib_utils.c

```diff
@@ -748,6 +746,21 @@ cib__signon_query(cib_t **cib, xmlNode **cib_object)
     }
 }
 
+int
+cib__clean_up_connection(cib_t *cib)
+{
+    int rc;
+
+    if (cib == NULL) {
+        return pcmk_rc_ok;
+    }
+
+    rc = cib->cmds->signoff(cib);
+    cib_delete(cib);
+    cib = NULL;
```

Comment:
You have to pass a pointer-pointer for this to be effective

Reply:
Oh yes, of course.  Fixed.